scripts/docs/features-html-output/build-features-html-output.js

```diff
@@ -0,0 +1,571 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const glob = require( 'glob' );
+const chalk = require( 'chalk' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+const DESTINATION_DOCS_PATH = 'docs/builds/guides/integration/features-html-output-overview.md';
+const THIRD_PARTY_PACKAGES_LOCAL_DIR = 'scripts/docs/features-html-output/third-party-packages';
+
+try {
+	const numberOfPackages = parseMetadataFiles();
+
+	console.log( `✨ ${ chalk.green( `The features HTML output from ${ numberOfPackages } packages has been generated successfully.` ) }` );
+
+	const shouldCommitChanges = process.argv.includes( '--commit' );
+
+	if ( shouldCommitChanges ) {
+		commitChanges();
+	}
+} catch ( error ) {
+	console.log( `❌ ${ chalk.red( 'An error occurred during parsing a package metadata file.' ) }` );
+	console.log( error );
```

Comment:
You miss here exiting with non-0 error code. Now, this script would pass even after the failure.

Reply:
Done.